src/algebra/ring/ulift.lean

```diff
@@ -21,6 +21,16 @@ variables {α : Type u} {x y : ulift.{v} α}
 
 namespace ulift
 
+@[simp] lemma cast_nat_down [add_monoid_with_one α] :
```

Comment:
This looks like `down_nat_cast` to me. Do we have `down_add` or `add_down` to set a convention?

Reply:
Also, isn't this true by rfl?

Reply:
> This looks like `down_nat_cast` to me. Do we have `down_add` or `add_down` to set a convention?

I followed the convention of [ulift.mul_down](https://leanprover-community.github.io/mathlib_docs/algebra/group/ulift.html#ulift.mul_down).

Reply:
> Also, isn't this true by rfl?

> It looks like this is about `ulift.add_monoid_with_one`, which I think is defined in an earlier file; can they be adjacent?

Indeed, both became possible with #12182.